### PR TITLE
translate overview and welcome page to Chinese

### PR DESCRIPTION
**Pull request by xieyuschen**
Hi, I have translated the `overview` and `welcome` pages to Chinese. 

Would I ask to you for writing permission from a branch so that I could manage the translation process for the Chinese version?

Thanks for your review and maintenance.

@gcolliso @matthiashanel

### CODE REVIEW

**Comment by xieyuschen** [comment]
Hi @matthiashanel , @gcolliso . Could you review it and how do you think the docs in Chinese?

**Comment by gcolliso** [comment]
@xieyuschen we don't have anyone on our team to review in Chinese. Would you have a co-worker or colleague that would be able to review?

**Comment by xieyuschen** [comment]
> @xieyuschen we don't have anyone on our team to review in Chinese. Would you have a co-worker or colleague that would be able to review?

Yes, it's my honor to work with you as a colleague to review the Chinese version.

**Comment by xieyuschen** [comment]
> @xieyuschen we don't have anyone on our team to review in Chinese. Would you have a co-worker or colleague that would be able to review?

yup, I would like to invite my friends to co-work with the docs.
Let me check with him.

**Comment by xieyuschen** [comment]
> @xieyuschen we don't have anyone on our team to review in Chinese. Would you have a co-worker or colleague that would be able to review?

To be honest, the work for documents might be boring especially in reviewing. 

I think we can also get help from the people who read our docs.

**Comment by FanHengbo** [comment]
Thanks for the invitation from @xieyuschen !
Hi @gcolliso , I think I might be able to review the Chinese translation.

File: zh-cn/overview.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+当今系统，如后端进程，微服务或者流式服务都要求可拓展性和安全性。  
+
+当前分布式系统正面临着从移动端向静态后端连接技术的挑战。当前技术通常利用1对1的通信模式，通过主机名（DNS）或者IP端口来寻址和发现服务，
+在此过程中也使用了很多不同的安全模式来认证和授权。虽然当前技术并不完美，但是差强人意，在大部分情况下表现足够好。但时境变迁，在微服务，
+函数以及流式处理技术在边缘设备的引进，当前的技术和假定遇到了很大的挑战。  
+
+
+## NATS关联技术与其他现代系统的区别是什么？
+
+**轻易联通M:N模型两端：** NATS的寻址与服务发现基于主题，而不是传统的主机名或者端口。默认采取1:1模型的超集，也就是M：N模型进行通信，
```
Comment (FanHengbo) [inline zh-cn/overview.md:19]: NATS的寻址与服务发现采用了基于主题的模式， 而并非像传统模式一样使用主机名和端口号

Comment (FanHengbo) [inline zh-cn/overview.md:19]:
```suggestion
**轻易联通M:N模型两端：** NATS的寻址与服务发现采用基于主题的模式，而不是像传统模式一样使用主机名和端口。默认采取1:1模型的超集，也就是M：N模型进行通信，
```

File: zh-cn/overview.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+这可以让他在1:1模型下做到更多的事情。如你在开发一个1:1模型的系统，请思考该系统需要多少额外的部分才可以绕开1:1的模型，如负载均衡器，日志系统
```
Comment (FanHengbo) [inline zh-cn/overview.md:20]:
```suggestion
这可以让它在1:1模型下做到更多的事情。例如你在开发一个1:1模型的系统，请思考该系统需要多少额外的部分才能在1:1模型的假设下运转，如负载均衡器，日志系统
```

File: zh-cn/overview.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+，网络安全模型，网络代理或者sidecar模式。如果你的系统需要以上所有去绕开这个模式，那么这也就是为什么要应用关联技术而不是原有技术，例如Http或者gRPC（都是1:1模型）的原因。
+是时候驻足于NATS.io了！
+
+**在任何地方部署：** NATS几乎可以在任何地方部署，无论是在裸机，虚拟机，容器里，还是在K8S集群，一台设备上都可以看到NATS的身影。无论在什么环境下，
+NATS可以良好工作，无论是否在部署框架下。
+
+**安全：** 相似的，NATS默认安全，不依赖于外部网络安全模型。很多时候，当你开始考虑后端微服务或者流式处理服务时，安全是一个相当重要的指标。
```
Comment (FanHengbo) [inline zh-cn/overview.md:27]:
```suggestion
**安全性：** 相似的，NATS在不依赖于外部网络安全模型的前提下是默认安全的，很多时候，当你开始考虑后端微服务或者流式处理服务时，安全是一个相当重要的指标。  
```

File: zh-cn/overview.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+### 混合部署
+使用NATS的另一个优势是它允许SaaS/实用程序计算与独占服务系统进行混合。这意味这你可以使用一个共享的NATS服务，在不同用户或者组织之间进行拓展其上的微服务，流式处理服务并允许每个人自己的NATS设施。
+不需要强制选择一个或者另一个方案。  
+
+### 适应性
+当今系统将无法满足新的需求。随着现代系统不断发展并利用更多组件和处理更多数据，支持1:1通信以外的模式以及与DNS相关的寻址和发现至关重要。像NATS这样的基础技术保证了最大的回报。
+随着现代系统，如融合云、边缘系统、物联网等，现有技术将无法发挥作用，但是NATS可以。
+
+### 使用场景
+NATS可以在任何地方允许，从大型服务器和云实例到边缘网关设备甚至物联网设备。NATS的使用场景包括：
```
Comment (FanHengbo) [inline zh-cn/overview.md:43]:
```suggestion
从大型服务器和云实例到边缘网关设备甚至物联网设备，NATS可以在任何地方运行。NATS的使用场景包括：
```

File: zh-cn/overview.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+* 云消息
+    * 服务（微服务，服务网络）
+    * 事件/数据流（观察，分析，加强学习与人工智能）
```
Comment (FanHengbo) [inline zh-cn/overview.md:47]:
```suggestion
    * 事件/数据流（观察，分析，机器学习/人工智能）
```

File: zh-cn/overview.md (hunk 2 above)
Comment (FanHengbo) [inline zh-cn/overview.md:20]:
```suggestion
这可以让它在1:1模型下做到更多的事情。如你在开发一个1:1模型的系统，请思考该系统需要多少额外的部分才可以绕开1:1的模型，如负载均衡器，日志系统
```

**Review by FanHengbo — CHANGES_REQUESTED** [review]
I've left some comments and suggestions that should be addressed before these two translations get merged.

**Comment by xieyuschen** [comment]
> I've left some comments and suggestions that should be addressed before these two translations get merged.

Fixed it, thanks.

**Review by FanHengbo — APPROVED** [review]

**Comment by xieyuschen** [comment]
@gcolliso I have invited my co-worker to review it, I think it is ok now.
Thanks

**Comment by Billdex** [comment]
Hello! I'm the author of issue #381 . I was pause the plan because of personal reason.
Today I try to restart translating work and notice this PR. I would join the translation.

Thanks for your work.

**Comment by xieyuschen** [comment]
> Hello! I'm the author of issue #381 . I was pause the plan because of personal reason. Today I try to restart translating work and notice this PR. I would join the translation.
> 
> Thanks for your work.


Thanks, welcome to work together.

**Pull request closed without merging**